src/console_reporter.cc

```diff
@@ -113,4 +121,16 @@ void ConsoleReporter::PrintRunData(const Run& result) {
               result.report_label.c_str());
 }
 
+TimeUnitMultiplier ConsoleReporter::getTimeUnitAndMultiplier(TimeUnit unit) {
```

Comment:
If i've asked a benchmark to report in ms, it should be reported in ms no matter which reporter I use.

as such, this can be moved to `BenchmarkReporter` and called from each of csv, json, and console reporters.